minard/polling.py

```diff
@@ -605,3 +605,30 @@ def get_vmon(crate, slot):
             bad_voltages[new_key] = 0 
 
     return result, bad_voltages
+
+def get_vmon_history(crate, slot):
+    """
+    Get the vmon history data for given crate/slot
+    """
+    conn = engine.connect()
+
+    voltages = str(voltages_str_dict.keys())[1:-1]
+    voltages = voltages.replace("'","")
+    query = "to_char(timestamp, 'Mon DD YYYY HH12:MI:SS'), "
+    query += voltages
+
+    result = conn.execute("SELECT %s FROM vmon WHERE crate = %s "
```

Comment:
Could you pass crate and slot as arguments. I think you'll probably need to create the query string first to avoid a string formatting error, i.e.

```python
 result = conn.execute("SELECT %s FROM vmon WHERE crate = %%s AND slot = %%s ORDER BY timestamp DESC" % query, crate slot)
```

or something similar.